docs/docs-components/COMPONENT_DATA.js

```diff
@@ -491,6 +493,7 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
   {
     svg: <Callout />,
     name: 'Callout',
+    aliases: ['Banner'],
```

Comment:
I know, this is more of a category name than a component name. Also this could conflict with new components we have planned. I'm just trying to meet users where they are, and we can always adjust the aliases in the future